apps/hostmetrics.go

```diff
@@ -301,6 +311,11 @@ func (r MetricsReceiverHostmetrics) Pipelines(_ context.Context) []otel.Receiver
 					otel.ToggleScalarDataType,
 					otel.AddLabel("process", "all"),
 				),
+				otel.RenameMetric(
```

Comment:
Should this also happen only on Windows?

Reply:
There should be no need, since it won't be enabled.

Reply:
Yeah it should be harmless to leave it here on both platforms, but might be confusing to future readers. The way this is written it's not elegant to exclude by just platform. What you'd need to do is change this to an array, add to the array conditionally based on platform, then spread it (with the `...` operator) into the arguments.

Reply:
It's confusing and pollutes the Linux configs, hence my suggestion.
As @braydonk said, you could do the following:
```
	transforms := []map[string]interface{}{
		// host transforms
		otel.RenameMetric(
			"system.cpu.time",
			"cpu/usage_time",
			…
		),
		…
		otel.RenameMetric(
			"system.paging.operations",
			"swap/io",
			…
		),
		// process transforms
		otel.RenameMetric(
			"process.cpu.time",
			"processes/cpu_time",
			…
		),
		…
		otel.RenameMetric(
			"process.memory.virtual",
			"processes/vm_usage",
			…
		),
	}
	if p.Type == platform.Windows {
		transforms = append(
			transforms, 
			otel.RenameMetric(
				"process.handles",
				"processes/windows/handles",
				otel.AddLabel("process", "all"),
			),
		)
	}
	transforms = append(
		transforms, 
		otel.AddPrefix("agent.googleapis.com"),
	)
	return []otel.ReceiverPipeline{{
		…
		Processors: map[string][]otel.Component{"metrics": {
			…
			otel.MetricsTransform(transforms...),
		}},
	}}
```